aws-rds-optiongroup/aws-rds-optiongroup.json

```diff
@@ -101,23 +101,7 @@
     },
     "EngineName": {
       "description": "Indicates the name of the engine that this option group can be applied to.",
```

Comment:
Can we then put the link to [official RDS documentation](https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_CreateOptionGroup.html ) for reference?

Reply:
Will reflect this in the internal tracker. Generally speaking, we should revert this change when CFN starts supporting case insensitive enums (or if). Considering non-blocking.